examples/kafka-hub/hub/Config.toml

```diff
@@ -48,5 +48,11 @@ cert.password = "password"
 key.keyStore.path = "./resources/brokercerts/client-keystore.jks"
 key.keyStore.password = "password"
 
+# The client MTLS configurations used by Kafka consumers mapped to HTTP subscribers
+[kafkaHub.config.KAFKA_CLIENT_KS_CONFIGS."test-consumer-group"]
+consumerGroupName="test-consumer-group"
+key.keyStore.path = "./resources/brokercerts/test-consumer-group-client-keystore.jks"
+key.keyStore.password = "password"
```

Comment:
We need to be able to give both keystore and turststore

Reply:
Why do we need the truststore ? They are connecting to the same kafka-cluster right ?